src/main/java/com/kodilla/ecommercee/repository/UserRepository.java

```diff
@@ -0,0 +1,19 @@
+package com.kodilla.ecommercee.repository;
+
+import com.kodilla.ecommercee.domain.User;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+public interface UserRepository extends CrudRepository<User, Long> {
+    List<User> findAll();
```

Comment:
I suggest useing JPARepository instead of crud like in GenericEntity so we dont have to implement standard crud methods

Reply:
> I suggest useing JPARepository instead of crud like in GenericEntity so we dont have to implement standard crud methods

I don't quite understand. As far as I can see, JpaRepository extends CrudRepository - it has a few additional functionalities (mainly related to sorting), but I don't know if we need it in any way, and that's pretty much of it (if I'm wrong - let me know).
I think this is one of those things we have to decide together, but in general I think it's better to use CrudRepository. One of many comparisons:
https://www.tutorialspoint.com/difference-between-crudrepository-and-jparepository-in-java

Reply:
I think we can use CrudRepository like we did with the Task project.

Reply:
I also use CrudRepository like in past projects.

Reply:
I have two CR here already (thx annapawlas and sew-k) and I could merge, but for now we have 3 votes for Crud, 1 for Jpa and two people with no comment.
I'll wait for one more support for Crud (or more details from JakubFir - maybe there is a good reason to use Jpa) before merging.

Reply:
Barosz confirmed Crud so I'm closing the discussion and merging. Thx for your input People...